sky/execution.py

```diff
@@ -533,8 +539,9 @@ def _translate_local_file_mounts(task: task_lib.Task) -> task_lib.Task:
     # Upload the local source to a bucket. The task will not be executed
     # locally, so we need to upload the files/folders to the bucket manually
     # here before sending the task to the remote spot controller.
-    logger.info(f'{colorama.Fore.YELLOW}Uploading sources to cloud storage.'
-                f'{colorama.Style.RESET_ALL} See sky storage ls')
+    if new_storage_mounts:
+        logger.info(f'{colorama.Fore.YELLOW}Uploading sources to cloud storage.'
+                    f'{colorama.Style.RESET_ALL} See: sky storage ls')
```

Comment:
This logging does not depend on the `new_storage_mounts`, as even without translation, users can still specify sky storages with source paths.

Reply:
Got it, I totally missed this due to the function name. Changed the function name a bit to avoid the confusion. Wdyt?